scripts/runAnalysisOverSamples.py

```diff
@@ -288,6 +289,7 @@ def CacheInputs(FileList):
                    if(doCacheInputs and isLocalSample):
                       result = CacheInputs(eventsFile)
                       eventsFile = result[0]
+                      LaunchOnCondor.Jobs_InitCmds.append('if [ -d $TMPDIR ] ; then cd $TMPDIR ; fi;\n')
```

Comment:
please add the if IIHE here
check if tmpdir exist might have very unexpected effect on other clusters